pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +46,24 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Add Current Profile Context
+func AddContextForProfile(profile string, ip string) error {
+	cmdName := instanceState.Config.OcPath
+	ip = strings.Replace(ip, ".", "-", -1)
+	cmdArgs := []string{"config", "set-context", profile,
+		fmt.Sprintf("--cluster=%s:%d", ip, constants.APIServerPort),
+		fmt.Sprintf("--user=developer/%s:%d", ip, constants.APIServerPort),
```

Comment:
Not matching with name of function since few things are hardcoded.
I would suggest to pass, `username` and `namespace` to this method and use those values. This will help to make this method general to use. 

Then, this method can be called as `AddContextForProfile("minishift", "some_ip", "user_name", "namespace")

WDYT ?